src/deadline/job_attachments/asset_sync.py

```diff
@@ -527,7 +528,16 @@ def _upload_output_manifest_to_s3(
             full_output_prefix,
             f"{manifest_name_prefix}_output",
         )
-        metadata = {"Metadata": {"asset-root": root_path}}
+        metadata = {"Metadata": {"asset-root": json.dumps(root_path, ensure_ascii=True)}}
```

Comment:
I will need to amend this to the `manifest upload` CLI since it has similar mechanisms.